app/helpers/report_helper/report_information_helper.rb

```diff
@@ -0,0 +1,108 @@
+      rows.push(row_data(_('Based On'), report_dictionary(report.db, :model)))
+      rows.push(row_data(_('User'), report.user ? report.user.userid : ""))
+      rows.push(row_data(_('EVM Group'), report.miq_group ? report.miq_group.description : ""))
+      rows.push(row_data(_('Updated On'), format_timezone(report.updated_on, Time.zone, "gtl")))
+      data[:rows] = rows
+      miq_structured_list(data)
+    end
+  end
+
+  def report_schedule(schedules)
+    data = {:title => _('Schedules'), :mode => "miq_report_schedules bordered-list simple_table", :rows => []}
+    if schedules.blank?
+      data[:message] = _("Report is not Scheduled.")
+    else
+      data[:headers] = [_('Name'), _('Description'), _('Active'), _('Interval'), _('Last Run Time'), _('Next Run Time'), _('Zone')]
+      data[:headers].push(_('Username')) if super_admin_user?
+
+      rows = []
+      schedules.each do |s|
+        tz = s.run_at && s.run_at[:tz] ? s.run_at[:tz] : session[:user_tz]
+        on_click = report_role('miq_report_schedules') ? on_click("msc-#{s.id}", "schedules") : ""
+        cells = [
+          {:value => s.name, :icon => "fa fa-clock-o"},
+          {:value => s.description},
+          {:value => _(s.enabled)},
+          {:value => s.run_at[:interval][:unit]},
+          {:value => s.last_run_on ? h(format_timezone(s.last_run_on, tz, "gtl")) : ""},
+          {:value => s.next_run_on ? h(format_timezone(s.next_run_on, tz, "gtl")) : ""},
+          {:value => s.v_zone_name},
+        ]
+        cells.push({:value => s.userid}) if super_admin_user?
+        rows.push({
+                    :cells   => cells,
+                    :title   => _("View this Schedule"),
+                    :onclick => on_click,
+                  })
+      end
+
+      data[:rows] = rows
+    end
+    miq_structured_list(data)
+  end
+
+  def report_widget(widget_nodes)
+    data = {:title => _('Widgets'), :mode => "miq_report_widgets bordered-list simple_table", :rows => []}
+    if widget_nodes.blank?
+      data[:message] = _("Report doesn't belong to Widgets.")
+    else
+      data[:headers] = [_('Title'), _('Description'), _('Enabled')]
+
+      rows = []
+      widget_nodes.each do |w|
+        on_click = report_role('miq_report_widget_editor') ? on_click("xx-#{ReportController::Widgets::WIDGET_CONTENT_TYPE.invert[w.content_type]}_-#{w.id}", "widgets") : ""
+        rows.push({
+                    :cells   => [
+                      {:value => h(w.title), :icon => "fa fa-file-text-o"},
+                      {:value => w.description},
+                      {:value => _(w.enabled)},
+                    ],
+                    :title   => _("Click to view selected widget"),
+                    :onclick => on_click,
+                  })
+      end
+
+      data[:rows] = rows
+    end
+    miq_structured_list(data)
+  end
+
+  def report_info_summary(sb_data, report, schedules, widget_nodes)
```

Comment:
Super minor, but anything that you aren't using in the view should probably be made into a private method.  This way we know they aren't called by anything else but the methods here.  So this method, I think, would be the only public method.  So, IMO, this one should be first in the file, followed by all the private methods.